app/mailers/application_mailer.rb

```diff
@@ -14,4 +20,36 @@ def global_config
   def smtp_config_set_or_development?
     ENV.fetch('SMTP_ADDRESS', nil).present? || Rails.env.development?
   end
+
+  def ensure_current_account(account)
```

Comment:
Are we using this function anywhere? I could not find a reference to this? Sorry if I missed it out somewhere.

Reply:
it was leftover initially but have repurposed the function.